graphs/ParkingProvider.json

```diff
@@ -0,0 +1,435 @@
+                "port": "outputdata"
+            },
+            "metadata": {
+                "route": 6
+            }
+        },
+        {
+            "src": {
+                "process": "Longitude",
+                "port": "out"
+            },
+            "tgt": {
+                "process": "flowhub-bigiot-bridge/CreateOffering_fjxkp",
+                "port": "outputdata"
+            },
+            "metadata": {
+                "route": 6
+            }
+        },
+        {
+            "src": {
+                "process": "NumberOfVacantSpaces",
+                "port": "out"
+            },
+            "tgt": {
+                "process": "flowhub-bigiot-bridge/CreateOffering_fjxkp",
+                "port": "outputdata"
+            },
+            "metadata": {
+                "route": 6
+            }
+        },
+        {
+            "src": {
+                "process": "RedisUrl",
+                "port": "out"
+            },
+            "tgt": {
+                "process": "ConnectRedis",
+                "port": "url"
+            },
+            "metadata": {
+                "route": 2
+            }
+        },
+        {
+            "src": {
+                "process": "ConnectRedis",
+                "port": "error"
+            },
+            "tgt": {
+                "process": "Errors",
+                "port": "in"
+            },
+            "metadata": {
+                "route": 1
+            }
+        },
+        {
+            "src": {
+                "process": "ConnectRedis",
+                "port": "client"
+            },
+            "tgt": {
+                "process": "flowhub-bigiot-bridge/GetParkingData_d15kz",
+                "port": "redis"
+            },
+            "metadata": {
+                "route": 2
+            }
+        },
+        {
+            "src": {
+                "process": "Provider",
+                "port": "request"
+            },
+            "tgt": {
+                "process": "flowhub-bigiot-bridge/GetParkingData_d15kz",
+                "port": "in"
+            },
+            "metadata": {
+                "route": 3
+            }
+        },
+        {
+            "src": {
+                "process": "flowhub-bigiot-bridge/GetParkingData_d15kz",
+                "port": "out"
+            },
+            "tgt": {
+                "process": "Provider",
+                "port": "response"
+            },
+            "metadata": {
+                "route": 4
+            }
+        },
+        {
+            "src": {
+                "process": "flowhub-bigiot-bridge/GetParkingData_d15kz",
+                "port": "error"
+            },
+            "tgt": {
+                "process": "Provider",
+                "port": "response"
+            },
+            "metadata": {
+                "route": 1
+            }
+        },
+        {
+            "data": "http://schema.org/parkingSiteManagement",
```

Comment:
I think this should be https://schema.big-iot.org/mobility/ParkingSiteStatus ?

It defines `parkingNumberOfOccupiedSpaces` in addition to `parkingNumberOfVacantSpaces`. So it would be nice for us to also have an estimate for this (subtract from total, which I do think we have)?
The `number` key in payload should maybe changed to something like `vacant` to clarify it